api/util/errorLog-function.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const router = express.Router();
+const ErrorLog = require('../models/ErrorLog');
+
+async function infoErrorLog(req){
```

Comment:
it would be helpful to add some documentation for what this function does (what the param is, what it returns)

see the [file you wrote](https://github.com/SCE-Development/Core-v4/blob/dev/src/APIFunctions/ErrorLog.js) for an example.